src/main/java/com/disi/TravelPoints/model/User.java

```diff
@@ -0,0 +1,85 @@
+package com.disi.TravelPoints.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "email")
+    private String email;
+    @Column(name = "password")
+    private String password;
+    @ManyToOne
+    @JsonIgnore
+    @JoinColumn(name="role_id", nullable = false)
+    private Role role;
+
+
+    public User() {}
+
+    public User(Long id, String name, String email, String password, Role role) {
+        this.id = id;
```

Comment:
Nu cred ca e nevoie de id in constructor, in mod normal ar trebui sa se asigneze unul in mod automat